README.md

```diff
@@ -73,19 +73,15 @@ type TransferArgs = record {
     to_principal: Principal;
     to_subaccount: opt SubAccount;
     amount: nat;
+    fee: opt nat;
+    memo: opt blob;
```

Comment:
Should be nat64

Reply:
Fixed in https://github.com/dfinity/ICRC-1/pull/19